aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -220,6 +220,9 @@ struct NormReduction {
         int64_t i = bi % stride;
         const scalar_t* data = &data_[b * n * stride + i];
         out_[bi] = norm_reduce(data, n, stride, pval);
+        if (!(pval == 0 || std::isinf(pval))) {
```

Comment:
you also need to add an std::pow in 203, I think? because reduce_all no longer does the pow(..., 1/pval)